core/migrations/0013_auto_20200511_1458.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.6 on 2020-05-11 14:58
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0012_auto_20200511_1449'),
```

Comment:
Because of the temporary rename, this won't be correct.  It should be `'0012_auto_20200507_1615'`

Reply:
I fixed it. Sorry about that